api/tests/Feature/SnapshotTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\PoolCandidate;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+use Nuwave\Lighthouse\Testing\ClearsSchemaCache;
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Illuminate\Foundation\Testing\WithFaker;
+
+use function PHPUnit\Framework\assertEquals;
+
+class SnapshotTest extends TestCase
+{
+    use RefreshDatabase;
+    use MakesGraphQLRequests;
+    use ClearsSchemaCache;
+    use WithFaker;
+
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function testCreateSnapshot()
+    {
+        $snapshotQuery = $query = file_get_contents(base_path('app/GraphQL/Mutations/TestPoolCandidateSnapshot.graphql'), true);
+
+        // set up a user with a pool candidate
+        $userId = $this->faker->Uuid();
+        $poolCandidateId = $this->faker->Uuid();
+        print_r($userId);
```

Comment:
Not a big deal, but I assume this `print_r` was just debugging and no longer needed?

Reply:
Good catch!  Fixed in [Remove stray print_r](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3707/commits/08750701ae9d71dec0159a2a9178c5bb4d5ddb1c)